pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
```

Comment:
Can you explain what this file gives us?

I understand that `pyproject.toml` files can replace `setup.py`, but I don't know why it's needed in this case?

Reply:
It was just from following the setuptools docs https://setuptools.readthedocs.io/en/latest/userguide/quickstart.html#basic-use

And just using `setup.py` I couldn't see an example of how to specify `wheel` as a build requirement. Happy to change though if its not necessary

Reply:
Interesting they're suggesting using toml files by default now! I don't mind it being here, but just want to make sure that this file isn't going to mess with `setup.cfg` in some way down the line. 

The other advantage of using this file is that we can configure `black` settings in it, for example. Let's keep it in and see if it changes anything (TBH, the whole python packaging thing is quite confusing for me!)

Reply:
> TBH, the whole python packaging thing is quite confusing for me!
Tell me about it!